# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/scribus/hotfix-lrelease.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- scribus-1.6.3/resources/translations/CMakeLists.txt.vanilla	2025-04-20 22:13:21.649438597 +0200
+++ scribus-1.6.3/resources/translations/CMakeLists.txt	2025-04-20 22:12:56.341136654 +0200
@@ -2,7 +2,7 @@
 "${CMAKE_SOURCE_DIR}/scribus"
 )
 
-set(LRELEASE_EXECUTABLE ${Qt5_LRELEASE_EXECUTABLE})
+set(LRELEASE_EXECUTABLE "lrelease")
 
 #Install our qm files, selected from *.qm in this directory
 file( GLOB SCRIBUS_TS_FILES ${CMAKE_CURRENT_SOURCE_DIR}/*.ts )
